apps/api/internal/data/gallery.go

```
@@ -19,17 +19,28 @@ type GalleryModel struct {
-		VALUES ($1, $2)
-		RETURNING id`
-
 	ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
 	defer cancel()
 
-	args := []any{img.ID, img.URL}
+	tx, err := m.DB.Begin(ctx)
+	if err != nil {
+		return err
+	}
+	defer tx.Rollback(ctx)
```

golangci-lint (2.12.2)

[error] Error return value of `tx.Rollback` is not checked

(errcheck)